pkg-config will only search the target sysroot and we want the native script. This 
patch adds an option to allow the path to the tool to be specified.

RP - 12/8/10

Upstream-Status: Pending

Index: sato-icon-theme-0.4.1/configure.ac
===================================================================
--- sato-icon-theme-0.4.1.orig/configure.ac	2010-08-12 12:23:25.000000000 +0100
+++ sato-icon-theme-0.4.1/configure.ac	2010-08-12 12:48:14.000000000 +0100
@@ -11,20 +11,29 @@
 
 UTILS_REQUIRED=0.8.2
 
-AC_MSG_CHECKING([icon-naming-utils >= $UTILS_REQUIRED])
-PKG_CHECK_EXISTS(icon-naming-utils >= $UTILS_REQUIRED,
-                 have_utils=yes, have_utils=no)
-if test "x$have_utils" = "xyes"; then
-   UTILS_PATH="`$PKG_CONFIG --variable=program_path icon-naming-utils`"
-   ICONMAP="$UTILS_PATH/icon-name-mapping"
-   AC_SUBST(ICONMAP)
-   AC_MSG_RESULT([yes])
-else
-   AC_MSG_RESULT([no])
-   AC_MSG_ERROR([icon-naming-utils >= $UTILS_REQUIRED is required to build
-                 and install sato-icon-theme])
+PKG_PROG_PKG_CONFIG()
+
+AC_ARG_WITH(iconmap,
+            AC_HELP_STRING([--with-iconmap=<dir>], [The location of the icon-name-mapping script to use]),
+            ICONMAP=$withval, ICONMAP="")
+
+if test "x$ICONMAP" = "x"; then
+   AC_MSG_CHECKING([icon-naming-utils >= $UTILS_REQUIRED])
+   PKG_CHECK_EXISTS(icon-naming-utils >= $UTILS_REQUIRED,
+                    have_utils=yes, have_utils=no)
+   if test "x$have_utils" = "xyes"; then
+      UTILS_PATH="`$PKG_CONFIG --variable=program_path icon-naming-utils`"
+      ICONMAP="$UTILS_PATH/icon-name-mapping"
+
+      AC_MSG_RESULT([yes])
+   else
+      AC_MSG_RESULT([no])
+      AC_MSG_ERROR([icon-naming-utils >= $UTILS_REQUIRED is required to build
+                    and install sato-icon-theme])
+   fi
 fi
 
+AC_SUBST(ICONMAP)
 
 AC_CONFIG_FILES([
 Makefile
